.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v2
       - name: Use Xcode 13
-        run: sudo xcode-select -switch /Applications/Xcode_13.0.app
+        run: sudo xcode-select -switch /Applications/Xcode_13.2.1.app
```

Comment:
Does something in this addition break on Xcode 13.0 or are we just updating to use the latest